src/amo/components/UserProfileEdit/index.js

```diff
@@ -610,9 +655,79 @@ export class UserProfileEditBase extends React.Component<Props, State> {
                     i18n.gettext("Update user's profile")
                 )}
               </Button>
+              <Button
+                buttonType="neutral"
+                className="UserProfileEdit-button UserProfileEdit-delete-button"
+                disabled={!user}
+                onClick={this.onDeleteProfile}
+                puffy
+                type="button"
+              >
+                {isEditingCurrentUser ?
+                    i18n.gettext('Delete my profile') :
+                    i18n.gettext(`Delete user's profile`)}
+              </Button>
             </div>
           </div>
         </form>
+
+        {this.state.showProfileDeletionModal && (
+          <OverlayCard
+            className="UserProfileEdit-deletion-modal"
+            header={isEditingCurrentUser ? i18n.gettext(
+              `Attention: you are about to delete your profile. Are you sure?`
```

Comment:
It seems like the "y" in "you" should be capitalized as it's the beginning of a sentence.

Reply:
Is it? I trust you on this because I often mix British, American and Broken English but I know that we don't always capitalize the first word after a colon.

Reply:
I don't actually know for sure, that was just an assumption on my part. Maybe we should ask @MeridelW for a final sign-off on a final set of screen shots?

Reply:
@bobsilverberg yes, I will open a new issue and assign @MeridelW to it :)